esrally/metrics.py

```diff
@@ -53,11 +55,13 @@ def search(self, index, doc_type, body):
 
     def guarded(self, target, *args, **kwargs):
         import elasticsearch
-        max_execution_count = 3
+        max_execution_count = 11
         execution_count = 0
 
         while execution_count < max_execution_count:
+            time_to_sleep = 2 ** execution_count + (random.randint(0, 1000) / 1000)
```

Comment:
Any reason you use `random.randint(0, 1000) / 1000` instead of just `random.random()`?

Reply:
Might also be worth adding a debug log statement?

Reply:
D'oh was thinking too much in milliseconds.